substrate/frame/identity/src/lib.rs

```diff
@@ -1005,8 +1005,8 @@ impl<T: Config> Pallet<T> {
 		Ok((new_id_deposit, new_subs_deposit))
 	}
 
-	/// Set an identity with zero deposit. Only used for benchmarking that involves `rejig_deposit`.
-	#[cfg(feature = "runtime-benchmarks")]
+	/// Set an identity with zero deposit. Used for benchmarking and XCM emulator tests, involves
+	/// `rejig_deposit`.
 	pub fn set_identity_no_deposit(
```

Comment:
I'm not crazy about exposing this `pub`, at least without a big warning. Is it possible to expose it only for tests?

Reply:
Added the feature `xcm-emulator`,
`#[cfg(any(feature = "xcm-emulator", feature = "runtime-benchmarks"))]`